webview-ui/src/components/settings/SettingsView.tsx

```diff
@@ -101,7 +101,9 @@ type SettingsViewProps = {
 
 const SettingsView = ({ onDone, targetSection }: SettingsViewProps) => {
 	// Track active tab
-	const [activeTab, setActiveTab] = useState<string>(targetSection || SETTINGS_TABS[0].id)
+	const [activeTab, setActiveTab] = useState<string>(
+		targetSection || (SETTINGS_TABS.find((tab) => tab.id === "general")?.id ?? SETTINGS_TABS[0].id),
```

Comment:
`SETTINGS_TABS.find((tab) => tab.id === "general")?.id` currently will always be true and therefore `?? SETTINGS_TABS[0].id` will be redundant and could cause unintended regression.

For https://github.com/cline/cline/issues/4705 passing "general" as the `targetSection` pro from webview-ui/src/App.tsx might be a safer option:

```ts
					{showSettings && (
						<SettingsView
							targetSection={telemetrySetting === "unset" ? "general" : undefined}
							onDone={hideSettings}
						/>
					)}
```